### PR TITLE
Displays background image if no job is displayed

### PR DESCRIPTION
**Pull request by oziks**

### CODE REVIEW

**Comment by KuiKui** [comment]
:+1:

File: app/styles/main.css [hunk 1]
```diff
@@ -11,6 +11,17 @@ a {
...
+#jenkinsLight {
+    min-height: 100%;
+    margin:0;
+    padding:0;
+    background-repeat: no-repeat;
+    background-position: center;
+    -webkit-background-size: cover; /* pour Chrome et Safari */
+    -moz-background-size: cover; /* pour Firefox */
+    -o-background-size: cover; /* pour Opera */
+    background-size: cover; /* version standardisée */
```
Comment (oltodo) [inline app/styles/main.css]: Sorry, I don't speak french :us:

Comment (oziks) [inline app/styles/main.css]: Roumain

**Comment by oltodo** [comment]
:+1:

**Comment by fdubost** [comment]
:+1:

**Pull request merged**